From cff343f18abd82f1e96a6ff090151a579e0929f6 Mon Sep 17 00:00:00 2001
From: Ky-Anh Huynh <kyanh@theslinux.org>
Date: Thu, 17 Oct 2013 21:51:27 +0700
Subject: [PATCH] PACKAGE_FEATURE will only alter $conflicts

Previously, the feature will alter $provides, $replaces.
However, $replaces is completely wrong. $provides may not
be always correct. $conflicts is enough. However, see
next commits and features
---
 pacman/src/pacman-4.1.0/scripts/makepkg.sh.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
index bc7957d..1d264d5 100644
--- a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
+++ b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
@@ -1818,8 +1818,6 @@ write_pkginfo() {
 
 	if [[ -n "$PACKAGE_FEATURE" ]]; then
 		conflicts=("${conflicts[@]}" "$pkgname")
-		provides=("${provides[@]}" "$pkgname")
-		replaces=("${replaces[@]}" "$pkgname")
 	fi
 
 	[[ $license ]]      && printf "license = %s\n"     "${license[@]}"
-- 
1.8.4

